### DISCUSSION TITLE
Print color to color?

### DISCUSSION THREAD

**Discussion by Ghostttie** [Textualize/rich#2638] [Q&A]
Good morning, just a little curious but is it possible to have a "color to color" type of thing which would look something like this when printed?
![image](https://user-images.githubusercontent.com/64477318/201240319-94121f2c-3e96-480b-92c1-38eea844b3a7.png)

Been looking over the docs for quite some time and can't seem to find a way to do this, any help is highly appreciated.

**Comment by geekya215** [comment]
I think you should try using [lolcat](https://github.com/busyloop/lolcat)
